web/frontend/src/components/slam-2d-render.vue

```diff
@@ -7,7 +7,26 @@ import * as THREE from 'three';
 import { MapControls } from 'three/examples/jsm/controls/OrbitControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
+import { SVGLoader } from 'three/examples/jsm/loaders/SVGLoader';
+import { baseMarkerUrl } from '../lib/base-marker-url'
+import { destMarkerUrl } from '../lib/destination-marker-url'
 
+type svgOffset = {
+  x: number,
+  z: number
+}
+
+const baseMarkerOffset: svgOffset = {
+  x: 0.35,
+  z: -0.55
```

Comment:
interesting to note that a negative value means moving "up" on the screen.
is this intended?

Reply:
this is a top down view of a world that is z positive forward, so think of the camera as first facing a z axis that is going in the negative direction, then rotating to face downward (-y). Positive z will indeed then be down and negative z will be up